common/src/main/scala/tsec/common/ManagedRandom.scala

```diff
@@ -13,7 +13,7 @@ trait ManagedRandom {
     * https://tersesystems.com/2015/12/17/the-right-way-to-use-securerandom/
     * Random may block on linux machiens
     */
-  private var cachedRand: SecureRandom = {
+  private[tsec] var cachedRand: SecureRandom = {
```

Comment:
this allows us to pass our cached rand in places where it takes the actual rand internally, then use `forceIncrement`